doc/user/cluster_tls.md

```diff
@@ -0,0 +1,73 @@
+# Cluster TLS Guide
+
+Cluster TLS policy is configured on a per-cluster basis via the TPR spec provided to etcd-operator.
+
+```yaml
+apiVersion: "etcd.coreos.com/v1beta1"
+kind: "Cluster"
+metadata:
+  name: "example-etcd-cluster"
+spec:
+  ...
+  clusterTLS:
+    ....
+```
+
+For a review etcd's TLS support and requirements, please read the relevant section from the [operations guide](https://coreos.com/etcd/docs/latest/op-guide/security.html).
+
+
+## Static Cluster TLS Policy
+
+```yaml
+apiVersion: "etcd.coreos.com/v1beta1"
+kind: "Cluster"
+metadata:
+  name: "example-etcd-cluster"
+ spec:
+  ...
+  clusterTLS:
+    static:
+      serverSecretName: server-tls-secret
+      clientSecretName: client-tls-secret
+
+```
+
+* **serverSecretName**: contains pem-encoded private keys and x509 certificates needed by the etcd server.
+
+  etcd-operator will mount this secret at `/etc/etcd-operator/server-tls` for each etcd member pod in the cluster.
+
+  The server TLS assets are expected to conform to the following structure:
+
+  ```text
+  /etc/etcd-operator/server-tls/
+
+       server-client-crt.pem
```

Comment:
> I feel like we should condense this discussion into a separate issue, TLS work is dependent on it but it's orthogonal. 

Since you're expecting users to generate these, you should be giving instructions about _what_ to generate. Is that orthogonal to documenting this feature? How would someone use this doc without that information?